tests/modules/ebi-metagenomics/checkm2/checkm2/test.yml

```diff
@@ -0,0 +1,21 @@
+- name: "checkm2 stub"
+  command: nextflow run ./tests/modules/ebi-metagenomics/checkm2/checkm2 -stub -entry test_checkm2 -c ./tests/config/nextflow.config -c ./tests/modules/ebi-metagenomics/checkm2/checkm2/nextflow.config
+  tags:
+    - "checkm2"
+    - "checkm2/checkm2"
+  files:
+    - path: "output/checkm2/test_checkm_output/quality_report.tsv"
+      md5sum: 32e53028a395debce647fa88da2261e5
+    - path: "output/checkm2/versions.yml"
```

Comment:
Remove all md5sum checks for versions.yml, because this will keep failing on container updates